internal/subscription/database/subscription_queries.go

```diff
@@ -94,5 +88,15 @@ func queriesSubscription() map[string]string {
 			WHERE subscriptions.user_uuid = $1 AND subscriptions.deleted_at IS NULL
 				AND matrices.deleted_at IS NULL
 				AND courses.deleted_at IS NULL`, returningColumns),
+		// UPDATE.
+		updateSubscription: fmt.Sprintf(`UPDATE subscriptions
```

Comment:
Updated doesn't need to `RETURNING %s` anymore.

Reply:
https://github.com/sumelms/microservice-course/pull/28/commits/0ba63413eecc914244b9b0cab324948e93aecaef